source/int_airflow.rst

```diff
@@ -0,0 +1,410 @@
+
+######################
+Airflow
+######################
+
+Legion provide a set of custom operators to give you the ability to interact with legion cluster using `Apache Airflow <https://airflow.apache.org/>`_
+
+
+***********************
+Connections
+***********************
+
+Airflow plugin should be authorized at Legion. Authorization is implemented using regular `Airflow Connections <https://airflow.apache.org/concepts.html#connections>`_
+
+All custom legion operators accept `edi_connection_id` as a parameter that refers to `Legion Connection`
+
+Legion Connection
+================
+The Legion connection provides access to a Legion cluster for Legion custom operators.
+
+Configuring the Connection
+--------------------------
+Host (required)
+    The host to connect to. Usually available at: `edi.<cluster-base-url>`
+
+Type (required)
+    `HTTP`
+
+Schema (optional)
+    `https`
+
+Login (required)
+    Specify the user name to connect.
+
+Password (required)
+    Specify the password to connect.
+
+Extra (optional)
+    Specify the extra parameters (as json dictionary) that can be used in Legion
+    connection. Because Legion uses OpenID authorization under the hood lots of extras are regular OpenID (OAuth 2.0)
```

Comment:
Because Legion uses OpenID authorization, additional OpenID/OAuth 2.0 parameters may be supplied here.